src/internal/connector/exchange/restore_test.go

```diff
@@ -266,9 +267,7 @@ func (suite *ExchangeRestoreSuite) TestRestoreExchangeObject() {
 			)
 			assert.NoError(t, err, support.ConnectorStackErrorTrace(err))
 			assert.NotNil(t, info, "item info is populated")
-
-			cleanupError := test.cleanupFunc(ctx, userID, destination)
-			assert.NoError(t, cleanupError)
+			assert.NoError(t, deleters[test.category].DeleteContainer(ctx, userID, destination))
```

Comment:
nit: if this is running as a cleanup function it should probably be run in a defer so we hit it in all cases